>>> 'Pkg.add("PyPlot")' log
INFO: Installing BinDeps v0.4.5
INFO: Installing ColorTypes v0.2.6
INFO: Installing Colors v0.6.7
INFO: Installing Conda v0.3.1
INFO: Installing FixedPointNumbers v0.1.6
INFO: Installing LaTeXStrings v0.2.0
INFO: Installing MacroTools v0.3.2
INFO: Installing PyCall v1.7.2
INFO: Installing PyPlot v2.2.2
INFO: Installing Reexport v0.0.3
INFO: Installing SHA v0.2.1
INFO: Installing URIParser v0.1.6
INFO: Building PyCall
INFO: No system-wide Python was found; got the following error:
could not spawn `'' -c "import distutils.sysconfig; print(distutils.sysconfig.get_config_var('VERSION'))"`: no such file or directory (ENOENT)
using the Python distribution in the Conda package
Traceback (most recent call last):
  File "/home/vagrant/.julia/v0.5/Conda/deps/usr/bin/conda", line 4, in <module>
    import conda.cli
  File "/home/vagrant/.julia/v0.5/Conda/deps/usr/lib/python2.7/site-packages/conda/cli/__init__.py", line 8, in <module>
    from .main import main  # NOQA
  File "/home/vagrant/.julia/v0.5/Conda/deps/usr/lib/python2.7/site-packages/conda/cli/main.py", line 46, in <module>
    from ..base.context import context
  File "/home/vagrant/.julia/v0.5/Conda/deps/usr/lib/python2.7/site-packages/conda/base/context.py", line 251, in <module>
    context = Context(SEARCH_PATH, conda, None)
  File "/home/vagrant/.julia/v0.5/Conda/deps/usr/lib/python2.7/site-packages/conda/common/configuration.py", line 692, in __init__
    self._add_search_path(search_path)
  File "/home/vagrant/.julia/v0.5/Conda/deps/usr/lib/python2.7/site-packages/conda/common/configuration.py", line 699, in _add_search_path
    return self._add_raw_data(load_file_configs(search_path))
  File "/home/vagrant/.julia/v0.5/Conda/deps/usr/lib/python2.7/site-packages/conda/common/configuration.py", line 371, in load_file_configs
    raw_data = odict(kv for kv in chain.from_iterable(load_paths))
  File "/home/vagrant/.julia/v0.5/Conda/deps/usr/lib/python2.7/collections.py", line 69, in __init__
    self.__update(*args, **kwds)
  File "/home/vagrant/.julia/v0.5/Conda/deps/usr/lib/python2.7/_abcoll.py", line 571, in update
    for key, value in other:
  File "/home/vagrant/.julia/v0.5/Conda/deps/usr/lib/python2.7/site-packages/conda/common/configuration.py", line 371, in <genexpr>
    raw_data = odict(kv for kv in chain.from_iterable(load_paths))
  File "/home/vagrant/.julia/v0.5/Conda/deps/usr/lib/python2.7/site-packages/conda/common/configuration.py", line 346, in _file_yaml_loader
    assert fullpath.endswith(".yml") or fullpath.endswith("condarc"), fullpath
AssertionError: /home/vagrant/.julia/v0.5/Conda/deps/usr/condarc-julia
===============================[ ERROR: PyCall ]================================

LoadError: failed process: Process(setenv(`/home/vagrant/.julia/v0.5/Conda/deps/usr/bin/conda install -y numpy`,String["PYTHON=","PATH=/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:/usr/games:/usr/local/games:/home/vagrant/julia/bin/","PWD=/vagrant/0.5MZ","SHELL=/bin/bash","SSH_CONNECTION=10.0.2.2 53039 10.0.2.15 22","LD_LIBRARY_PATH=:/usr/local/lib/","CONDARC=/home/vagrant/.julia/v0.5/Conda/deps/usr/condarc-julia","MAIL=/var/mail/vagrant","LANG=en_US.UTF-8","SHLVL=3","LOGNAME=vagrant","XDG_RUNTIME_DIR=/run/user/1000","XDG_SESSION_ID=3","OLDPWD=/home/vagrant","SSH_CLIENT=10.0.2.2 53039 22","_=/usr/bin/timeout","USER=vagrant","TERM=vt100","HOME=/home/vagrant","OPENBLAS_MAIN_FREE=1","JAVA_HOME=/usr/lib/jvm/java-7-oracle","PYTHONIOENCODING=UTF-8"]), ProcessExited(1)) [1]
while loading /home/vagrant/.julia/v0.5/PyCall/deps/build.jl, in expression starting on line 17

================================================================================

================================[ BUILD ERRORS ]================================

WARNING: PyCall had build errors.

 - packages with build errors remain installed in /home/vagrant/.julia/v0.5
 - build the package(s) and all dependencies with `Pkg.build("PyCall")`
 - build a single package by running its `deps/build.jl` script

================================================================================
INFO: Package database updated

>>> 'Pkg.test("PyPlot")' log
Julia Version 0.5.0-rc4+0
Commit 9c76c3e (2016-09-09 01:43 UTC)
Platform Info:
  System: Linux (x86_64-pc-linux-gnu)
  CPU: Intel(R) Xeon(R) CPU E3-1241 v3 @ 3.50GHz
  WORD_SIZE: 64
           Ubuntu 14.04.5 LTS
  uname: Linux 3.13.0-95-generic #142-Ubuntu SMP Fri Aug 12 17:00:09 UTC 2016 x86_64 x86_64
Memory: 2.4471054077148438 GB (544.046875 MB free)
Uptime: 14096.0 sec
Load Avg:  1.1103515625  0.95166015625  0.9033203125
Intel(R) Xeon(R) CPU E3-1241 v3 @ 3.50GHz: 
       speed         user         nice          sys         idle          irq
#1  3500 MHz     741052 s        108 s      92275 s     389969 s         18 s
#2  3500 MHz     224816 s       5608 s      40933 s    1085151 s          0 s

  BLAS: libopenblas (USE64BITINT DYNAMIC_ARCH NO_AFFINITY Nehalem)
  LAPACK: libopenblas64_
  LIBM: libopenlibm
  LLVM: libLLVM-3.7.1 (ORCJIT, haswell)
Environment:
  LD_LIBRARY_PATH = :/usr/local/lib/
  HOME = /home/vagrant
  TERM = vt100
  PATH = /usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:/usr/games:/usr/local/games:/home/vagrant/julia/bin/
  JAVA_HOME = /usr/lib/jvm/java-7-oracle

Package Directory: /home/vagrant/.julia/v0.5
2 required packages:
 - JSON                          0.7.0
 - PyPlot                        2.2.2
12 additional packages:
 - BinDeps                       0.4.5
 - ColorTypes                    0.2.6
 - Colors                        0.6.7
 - Compat                        0.9.2
 - Conda                         0.3.1
 - FixedPointNumbers             0.1.6
 - LaTeXStrings                  0.2.0
 - MacroTools                    0.3.2
 - PyCall                        1.7.2
 - Reexport                      0.0.3
 - SHA                           0.2.1
 - URIParser                     0.1.6
INFO: Testing PyPlot
INFO: PyPlot tests passed

>>> End of log
